server/router.js

```diff
@@ -31,7 +32,12 @@ router.on('/:archiveKey', {
   get: function (req, res, params) {
     let state = getDefaultAppState()
     state.archive.key = params.archiveKey
-    sendSPA('/:archiveKey', res, state)
+    var archive = haus(state.archive.key)
+    archive.list(function (err, data) {
+      if (err) throw err
```

Comment:
`if (err)`: instead of just throwing an error here we should probably add an error model to the app with the correct messaging and go ahead and render the page via sendSPA with whatever message is defined by UX flow mocks. I'm fine for this PR not implementing the full error message design as defined in the UX flow issue, but we should probably not just throw an error every time an archive drops out of swarm(s).